modules/oe_translation_cdt/tests/fixtures/cdt-source-file.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Transaction xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="//static.cdt.europa.eu/webtranslations/schemas/Drupal-Translation-V8-2.xsd">
+  <TransactionHeader>
+    <SenderDetails drupalVersion="@drupal_version" moduleVersion="8.x-1.6">
```

Comment:
The parameters starting with `@` will be changed later by a simple `str_replace` inside a test.

I generate this file in the same way as `tmgmt_cdt` module does.